install_files/ansible-base/roles/ossec/templates/send_encrypted_alarm.sh

```diff
@@ -16,34 +16,56 @@ set -o pipefail
 # encryption fail.
 ossec_alert_text="$(< /dev/stdin)"
 
+# default to environment even if null
+env | grep -q JOURNALIST_EMAIL || JOURNALIST_EMAIL='{{ journalist_alert_email }}'
+env | grep -q OSSEC_EMAIL || OSSEC_EMAIL='{{ ossec_alert_email }}'
+
 # Primary "send email to Admin" functionality.
 function send_encrypted_alert() {
+    local recipient="$1"
+    local gpg_fpr
+    local alert_email
+    local compression
+    if [[ "$recipient" = "journalist" ]] ; then
+        gpg_fpr='{{ journalist_gpg_fpr }}'
+        alert_email="$JOURNALIST_EMAIL"
+        if [[ "$alert_email" = "" ]] ; then
+            echo "journalist alert email unset, no notification sent"
+            return
+        fi
+        compression="--compress-algo none"
```

Comment:
I had to modify this to `compression="--compress-algo=none"` or I get a failure to encrypt message

Reply:
Hmm, also the failure message went to the journalist and not the admin - I think this is ok (otherwise the journalist might wonder "where's my daily email") but we could add a "Please contact your administrator" string

Reply:
> I had to modify this to ...

Ouch. It means the tests do not run this code path for some reason... I'll look into that.

Reply:
I didn't encounter this problem; @redshiftzero do you still recommend the change here?

Reply:
I have _not_ run on the latest changes here, but when I last tested I did have to make this change (to generate emails for my compression vs no compression testing last month). I would need to check again in VMs and verify it's no longer the case... I can look into this on Monday

Reply:
Ok, just re-ran on the latest, and things are indeed working as is, so this change is not needed (not 100% sure what was happening before, but since I made some modifications to test the ciphertext lengths it's possible I introduced an unrelated issue).